### PR TITLE
Page content container

### PR DESCRIPTION
**Pull request by Vomcat**

### CODE REVIEW

**Review by TomaszSack — APPROVED** [review]

File: src/components/ui/PageContentContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+type Props = {
+  children: React.ReactNode;
+};
+
+const Wrapper = styled.div`
+  padding: 0 16px;
+
+  ${respondFrom(breakpoints.desktop)`
+      max-width: 1232px;
```
Comment (magnowak) [inline src/components/ui/PageContentContainer.tsx]: 1232 looks like a magic  number. I would suggest creating a global const (e.g. PAGE_CONTENT_WIDTH = 1232) at the top of this file and then using it here (in a template string). it's quite probable that you will be using this value in other places

File: src/components/ui/PageContentContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+
+import { respondFrom, breakpoints } from "../../styles";
+
+type Props = {
+  children: React.ReactNode;
+};
+
+const Wrapper = styled.div`
+  padding: 0 16px;
```
Comment (magnowak) [inline src/components/ui/PageContentContainer.tsx]: I would also consider creating global spacing definition and referring to it here. Your spacing value will probably be 8px

Comment (magnowak) [reply src/components/ui/PageContentContainer.tsx]: this way you will have a consistent basic value for defining paddings, margins, etc

File: src/components/ui/PageContentContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+
+import { respondFrom, breakpoints } from "../../styles";
```
Comment (magnowak) [inline src/components/ui/PageContentContainer.tsx]: it is recommended to use absolute paths. it's safer that way as it avoids wrong path resolution

**Review by magnowak — APPROVED** [review]
Looks good minor improvements suggested.

File: src/components/ui/PageContentContainer.tsx (hunk 2 above)
Comment (Vomcat) [reply src/components/ui/PageContentContainer.tsx]: I add new variables for spacing

File: src/components/ui/PageContentContainer.tsx (hunk 1 above)
Comment (Vomcat) [reply src/components/ui/PageContentContainer.tsx]: I create new variable for content page with

File: src/components/ui/PageContentContainer.tsx (hunk 3 above)
Comment (Vomcat) [reply src/components/ui/PageContentContainer.tsx]: Ok, i change it

**Pull request merged**